doc/figures/evaluate_sequence.svg

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The worker 1 only get the evaluation the task but doesn't process it.
It will be better to add this process.